%YAML:1.0

left: 
    K: !!opencv-matrix
        rows: 3
        cols: 3
        dt: d
        data: [458.654880721, 0.0, 367.215803962,
               0.0, 457.296696463, 248.37534061,
               0.0, 0.0, 1.0]
    D: !!opencv-matrix
        rows: 1
        cols: 4
        dt: d
        data: [-0.28340811217, 0.0739590738929, 0.000193595028569, 1.76187114545e-05]

right:
    K: !!opencv-matrix
        rows: 3
        cols: 3
        dt: d
        data: [457.587426604, 0.0, 379.99944652,
               0.0, 456.13442556, 255.238185386,
               0.0, 0.0, 1.0]
    D: !!opencv-matrix
        rows: 1
        cols: 4
        dt: d
        data: [-0.283683654496, 0.0745128430929, -0.000104738949098, -3.55590700274e-05]

image_width: 752
image_height: 480